includes/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to WooCommerce Payments Core
+
+The [main readme file](README.md) in this directory outlines the concept of WooCommerce Payments' core. Please familiarize yourself with it before proceeding with this document.
+
+## Existing and additional core functionality
+
+The initial (and so far current) contents of this directory, as described in README.md is the outcome of the initial phase of the WooCommerce Payments as a Platform project. However, in addition to further phases of the project, further contributions to the plugin should also strive to follow the same principles.
+
+### Services
+
+There are a few possible paths when it comes to services:
+
+1. __Create a facade for an existing service:__ Create a new service class within `core/services`, which simply facades the existing service. Doing so will allow us to modify the facade in the future, keeping existing methods with the same parameters as existing ones.
```

Comment:
Initially this was not linked, because there was a link to the service on the next line, but I added it to https://github.com/Automattic/woocommerce-payments/pull/5608/commits/f02f8a4d200e2109be4991d0e4678a6ed24c6910 because why not :)

As for examples, I think the customer service speaks more than enough: https://github.com/Automattic/woocommerce-payments/blob/f02f8a4d200e2109be4991d0e4678a6ed24c6910/includes/core/service/class-wc-payments-customer-service-api.php#L1